modules/pixfutureBidAdapter.js

```diff
@@ -0,0 +1,319 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { getStorageManager } from '../src/storageManager.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { config } from '../src/config.js';
+import includes from 'core-js-pure/features/array/includes.js';
+import * as utils from '../src/utils.js';
+import { auctionManager } from '../src/auctionManager.js';
+import find from 'core-js-pure/features/array/find.js';
+
+const SOURCE = 'pbjs';
+const storageManager = getStorageManager();
+const USER_PARAMS = ['age', 'externalUid', 'segments', 'gender', 'dnt', 'language'];
+export const spec = {
+  code: 'pixfuture',
+  hostname: 'https://prebid-js.pixfuture.com',
+
+  getHostname() {
+    let ret = this.hostname;
+    try {
+      ret = storageManager.getDataFromLocalStorage('ov_pixbidder_host') || ret;
+    } catch (e) {
+    }
+    return ret;
+  },
+
+  isBidRequestValid(bid) {
+    return !!(bid.sizes && bid.bidId && bid.params &&
+                (bid.params.pix_id && (typeof bid.params.pix_id === 'string')));
+  },
+
+  buildRequests(validBidRequests, bidderRequest) {
+    const tags = validBidRequests.map(bidToTag);
+    const hostname = this.getHostname();
+    return validBidRequests.map((bidRequest) => {
+      let referer = '';
+      if (bidderRequest && bidderRequest.refererInfo) {
+        referer = bidderRequest.refererInfo.referer || '';
+      }
+
+      const userObjBid = find(validBidRequests, hasUserInfo);
+      let userObj = {};
+      if (config.getConfig('coppa') === true) {
+        userObj = {'coppa': true};
```

Comment:
You didn't add `coppa` support in your docs PR.

Reply:
The `coppa` has been added to docs.
Also added added `meta.advertiserDomains` to the pixfutureBidAdapter.js